File: stock-market-service/src/main/java/rs/edu/raf/dto/GlobalQuoteDTO.java
package rs.edu.raf.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public record GlobalQuoteDTO(
        @JsonProperty("03. high")
        Double high,
        @JsonProperty("04. low")
        Double low,
        @JsonProperty("05. price")
        Double price,
        @JsonProperty("06. volume")
        Long volume,
        @JsonProperty("09. change")
        Double change
) {
}

File: stock-market-service-kotlin/src/main/kotlin/org/example/stockmarketservicekotlin/dto/GlobalQuoteDTO.kt
package org.example.stockmarketservicekotlin.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class GlobalQuoteDTO(
    @JsonProperty("03. high")
    val high: Double,
    @JsonProperty("04. low")
    val low: Double,
    @JsonProperty("05. price")
    val price: Double,
    @JsonProperty("06. volume")
    val volume: Long,
    @JsonProperty("09. change")
    val change: Double
)
